frontend/ui/widget/inputtext.lua

```diff
@@ -664,6 +669,18 @@ function InputText:onKeyPress(key)
             handled = false
         end
     end
+    if not handled and Device:hasSymKey() then
+        local symkey = sym_key_map[key.key]
+        if symkey then
+            for modifier, flag in pairs(key.modifiers) do
+                if modifier == "Sym" and flag then
+                    self:addChars(symkey)
+                end
+            end
```

Comment:
I see this is not all your making - there's another existing bit elsewhere that does loop key.modifiers.
But why doesn't a simpler
`if symkey and key.modifiers["Sym"] then self:addChars(symkey) end` would not work?

Reply:
> But why doesn't a simpler … work

Well that was very much my point, the for loop seemed a bit too much and i was asking if you had a simpler method that could do the same (perhaps wrongly conveyed, it is difficult to only communicate via text) ;) i’ll try it in a bit